lang/py/src/avro/schema.py

```diff
@@ -551,6 +506,69 @@ def to_json(self, names=None):
   def __eq__(self, that):
     return self.props == that.props
 
+#
+# Decimal Fixed Type
+#
+
+class FixedDecimalSchema(FixedSchema, LogicalSchema):
+  def __init__(self, schema_type, size, name, precision, scale=0, namespace=None, names=None, other_props=None):
```

Comment:
It doesn't need to take `schema_type` as parameter if it already knows it is `FixedSchema`.